### PR TITLE
Add switch for variant stats array version

### PR DESCRIPTION
**Pull request by awenocur**

### CODE REVIEW

**Review by gspowley — APPROVED** [review]
Looks good! 

A couple of comments about the sample buffer for v2.

File: libtiledbvcf/src/stats/variant_stats.cc [hunk 1]
```diff
@@ -341,10 +438,16 @@ void VariantStats::flush(bool finalize) {
     contig_buffer_.clear();
     contig_offsets_.clear();
     pos_buffer_.clear();
+    sample_buffer_.clear();
+    sample_offsets_.clear();
```
[the remark this answers is not in the document]
Comment (awenocur) [reply libtiledbvcf/src/stats/variant_stats.cc:442]: yes; they're unused for v2

File: apis/python/src/tiledbvcf/binding/writer.h [hunk 2]
```diff
@@ -245,6 +245,11 @@ class Writer {
   */
   void set_compression_level(int level);
 
+  /**
+  Set zstd compression level
```
[the remark this answers is not in the document]
Comment (awenocur) [reply apis/python/src/tiledbvcf/binding/writer.h:249]: whoops!

[the remark this answers is not in the document]
Comment (awenocur) [reply apis/python/src/tiledbvcf/binding/writer.h]: fixed

**Pull request merged**